src/test/scala/org/broadinstitute/dsde/firecloud/service/EntityServiceSpec.scala

```diff
@@ -241,27 +229,36 @@ class EntityServiceSpec extends BaseServiceSpec with EntityService {
       }
     }
 
-    "when calling bulk entity delete" - {
-      "response is OK" ignore {
+    "when calling bulk entity delete with a valid payload" - {
```

Comment:
I am skeptical of how useful mock tests are, but these were easy enough to write so :man_shrugging: :woman_shrugging: